lib/ui/screen/const/value_text.dart

```diff
@@ -13,6 +14,15 @@ class AppTexts {
   static const String textVisited = 'Посетил';
   static const String textWantToVisit = 'Хочу посетить';
   static const String title = 'Flutter Demo';
+
+  static const String textCategories = 'Категории';
+  static const String textDistance = 'Расстояние';
+
+  static const String textClear = 'Очистить';
+  static const String textShow = 'Показать';
+
+  static const String textSettingsTheme = 'Темная тема';
+  static const String textSettingsInfo = 'Смотреть туториал';
 }
 
 const mapSightTypeToString = {

```

Comment:
#### enum
https://medium.com/codex/flutter-3-what-are-enums-and-what-is-new-about-it-f5f2c481e7b5

```dart
enum PizzaSize {
  S(name: PizzaStrings.sizeS, size: 25),
  M(name: PizzaStrings.sizeM, size: 30),
  L(name: PizzaStrings.sizeL, size: 35);

  final String name;
  final int size;

  const PizzaSize({
    required this.name,
    required this.size,
  });
}


class PizzaStrings {
  /// Значения для enum
  static const sizeS = 'Маленькая';
  static const sizeM = 'Средняя';
  static const sizeL = 'Большая';

  PizzaStrings._();
}

```